.github/workflows/autobump.yml

```diff
@@ -1362,9 +45,13 @@ jobs:
         with:
           username: ${{ (github.event_name == 'workflow_dispatch' && github.actor) || 'BrewTestBot' }}
 
+      - name: Get list of autobump formulae
+        id: autobump
+        run: echo "autobump_list=$(< "$GITHUB_WORKSPACE"/autobump.txt tr '\n' ' ')" >> "$GITHUB_OUTPUT"
```

Comment:
```suggestion
        run: cat "$GITHUB_WORKSPACE"/.github/workflows/autobump.txt" | xargs >> "$GITHUB_OUTPUT"
```

Reply:
Shellcheck complained about UUOC, https://stackoverflow.com/questions/11710552/useless-use-of-cat, and made me change it to this syntax.

Reply:
Were you using `xargs` before?

Reply:
Alright, makes sense. It was because I was using `tr` that I got the recommendation. But `xargs` looks easier to understand. I changed it to `xargs`.